library-extensions-expandable/src/main/java/com/mikepenz/fastadapter/expandable/ExpandableExtension.kt

```diff
@@ -22,13 +24,36 @@ fun <Item : GenericItem> FastAdapter<Item>.getExpandableExtension(): ExpandableE
  * Extension method to retrieve or create the ExpandableExtension from the current FastAdapter
  * This will return a non null variant and fail
  */
-fun <Item : GenericItem> FastAdapter<Item>.expandableExtension(block: ExpandableExtension<Item>.() -> Unit) {
+inline fun <Item : GenericItem> FastAdapter<Item>.expandableExtension(block: ExpandableExtension<Item>.() -> Unit) {
     getExpandableExtension().apply(block)
 }
 
+/**
+ * internal helper function to check if an item is expanded.
+ */
+val IItem<out RecyclerView.ViewHolder>?.isExpanded: Boolean
```

Comment:
@mikepenz wait so did you decide for this to be internal or not? This is currently public

Reply:
As per your previous comment, I'm not sure why an internal val would not be accessible in the module from which it is declared

Reply:
oops the internal got lost with me changing it to the property. You are absolutely right. will address this.